package.json

```diff
@@ -42,6 +42,13 @@
     "typescript": "2.0.3"
   },
   "devDependencies": {
+    "@types/fs-extra": "0.0.33",
+    "@types/handlebars": "^4.0.31",
+    "@types/highlight.js": "^9.1.8",
+    "@types/lodash": "^4.14.37",
+    "@types/marked": "0.0.28",
+    "@types/minimatch": "^2.0.29",
+    "@types/shelljs": "^0.3.32",
```

Comment:
done. 

I'm not 100% sure this is the right thing to do though.  I think we only need to add as `dependencies` those @types/* packages which are exposed via a public API in the typedoc.d.ts files.